source/_integrations/xiaomi_miio.markdown

```diff
@@ -656,31 +744,6 @@ Set the fan operation mode.
 | `entity_id`               |       no | Only act on a specific Xiaomi miIO fan entity.                      |
 | `preset_mode`             |       no | The Xiaomi miIO operation mode                                      |
 
-### Service `switch.turn_on` (Air Humidifiers only)
```

Comment:
I removed the information about the `switch`, `number` and `select` platform services. This is platform specific information and in my opinion we shouldn't duplicate it here.